commands/src/main/java/org/wildfly/extras/creaper/commands/logging/AddLogCategory.java

```diff
@@ -0,0 +1,104 @@
+package org.wildfly.extras.creaper.commands.logging;
+
+
+import org.wildfly.extras.creaper.commands.foundation.offline.xml.GroovyXmlTransform;
+import org.wildfly.extras.creaper.commands.foundation.offline.xml.Subtree;
+import org.wildfly.extras.creaper.core.CommandFailedException;
+import org.wildfly.extras.creaper.core.offline.OfflineCommandContext;
+import org.wildfly.extras.creaper.core.online.OnlineCommandContext;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.Batch;
+import org.wildfly.extras.creaper.core.online.operations.OperationException;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+import org.wildfly.extras.creaper.core.online.operations.Values;
+
+/**
+ * <p>
+ * Command adds log category.
```

Comment:
This comment only replicates the class name. If there's nothing more to say than the class name, just avoid the comment completely.